dev/com.ibm.ws.transport.http/src/io/openliberty/http/options/CompressionOption.java

```diff
@@ -55,10 +55,4 @@ public Class<?> getValueType() {
     public ConfigType getConfigType() {
         return configType;
     }
-
```

Comment:
Copyright date range update? Seems significant enough?

Reply:
There are a number of classes with similar updates, if you don't agree that's fine just let me know. I figure since we're changing functionality and no longer overriding a method that to me is fairly significant.

Reply:
I'll make the update to the copyright